app/src/main/java/com/google/samples/apps/sunflower/data/GardenPlantingRepository.kt

```diff
@@ -16,23 +16,17 @@
 
 package com.google.samples.apps.sunflower.data
 
-import com.google.samples.apps.sunflower.utilities.runOnIoThread
-
 class GardenPlantingRepository private constructor(
     private val gardenPlantingDao: GardenPlantingDao
 ) {
 
     fun createGardenPlanting(plantId: String) {
-        runOnIoThread {
-            val gardenPlanting = GardenPlanting(plantId)
-            gardenPlantingDao.insertGardenPlanting(gardenPlanting)
-        }
+        val gardenPlanting = GardenPlanting(plantId)
```

Comment:
`withContext(IO)` to allow this to be called from any dispatcher. This will require making this a suspend function.